libs/common/src/types/index.ts

```diff
@@ -23,7 +23,7 @@ export * from './api-error-code';
 export * from './facility-type';
 export * from './deal-type';
 export * from './any-object';
-export * from './tfm-deal-cancellation';
+export * from './deal-cancellation';
```

Comment:
What's the reason for the rename please, deal cancellation is TFM specific?

Reply:
Oops, I did this when changing the name of the type and forgot to put it back